cluster-scope/overlays/nerc-ocp-prod/ingresscontrollers/external-apps-ingress-controller.yaml

```diff
@@ -15,6 +15,6 @@ spec:
     nodeSelector:
       matchLabels:
         nerc.mghpcc.org/external-ingress: 'true'
-  namespaceSelector:
+  routeSelector:
```

Comment:
We need to add the `domain` option **here** (ignore that earlier comment if you saw it; I've remove it). Relevant docs are https://docs.openshift.com/container-platform/4.10/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.html#nw-ingress-sharding-route-labels_configuring-ingress-cluster-traffic-ingress-controller.

We should set domain to `apps.shift.nerc.mghpcc.org`.